PersonApi.Tests/V1/ExceptionMiddlewareTests.cs

```diff
@@ -0,0 +1,92 @@
+using FluentAssertions;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Primitives;
+using Moq;
+using NUnit.Framework;
+using PersonApi.V1;
+using PersonApi.V1.Controllers;
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Threading.Tasks;
+
+namespace PersonApi.Tests.V1
+{
+    [TestFixture]
+    public class ExceptionMiddlewareTests
+    {
+        private readonly string _traceId = Guid.NewGuid().ToString();
+        private readonly string _correlationId = Guid.NewGuid().ToString();
+        private const int DEFAULTERRORCODE = 500;
+        private const string DEFAULTERRORMESSAGE = "Internal Server Error.";
+
+        private HttpContext _httpContext;
+        private Mock<IExceptionHandlerFeature> _mockExHandlerFeature;
+        private readonly InMemoryTraceListener _traceListener = new InMemoryTraceListener();
+
+        [SetUp]
```

Comment:
I wonder if we should also move this to Xunit, given they are newly written tests.

Reply:
Rocco suggested we just start with xUnit in the search service, so I kept to using nUnit. We can do a wholesale migration as a small separate ticket afterwards.